include/swift/SwiftRemoteMirror/SwiftRemoteMirrorTypes.h

```diff
@@ -48,11 +48,22 @@ typedef struct swift_reflection_section {
   void *End;
 } swift_reflection_section_t;
 
+/// Represents the remote address and size of an image.
+typedef struct swift_remote_reflection_section {
```

Comment:
Is this actually a section as per the file format or is this really just the specific metadata table?

Reply:
This is supposed to represent the counterpart of the already existing `swift_reflection_section` struct. That's why I named it `swift_remote_reflection_section`. Is this name wrong? Aren't `associated_types`, `builtin_types`, `capture`, etc sections?

Reply:
They are sections only on ELF because of the restrictions from the file format.  Each one of those is a separate metadata table.  In fact, if you look at MachO, they are all in the `__DATA` section and each is a segment.

Reply:
You have that backwards, `__DATA` is the segment (they're actually in `__TEXT`, though, because they're constant), and `__TEXT, __swift5_foo` is the section, in Mach-O parlance. The name seems correct to me.

Reply:
Ah, okay, if that is constant across all the formats, sounds reasonable.